src/localization/locales/en.json

```
@@ -183,6 +193,11 @@
       "clickToPreview": "Click to preview",
       "addButton": "Add new attachment",
       "uploadDialogTitle": "Add a new attachment",
+      "uploadExistingFileConfirmationDialog": {
+        "title": "Confirm existing file upload",
+        "description": "The file you are trying to upload already exists on the server storage. It will be reused for this task. Are you sure you want to proceed?",
```

This could also mention, that you can upload the file you want by changing it's name.

```suggestion
        "description": "A file with the same name you are trying to upload already exists. Do you want to reuse the already existing file? You can also change the name of the file and upload it again.",
```